tests/run.sh

```diff
@@ -103,7 +103,7 @@ wait_until_success() {
     if [ "$CI" = true ]; then
         set +x
     fi
-    max_wait=60
+    max_wait=30
```

Comment:
This is the maximum number of loop iteration, not the number of seconds to wait. In the loop you will see a `sleep 0.05`, so the old time is `0.05s * 60 = 3 seconds` while the new time is 1.5 seconds. Not much of a difference. (The commit message talks about seconds)